src/app/navbar/navbar.component.ts

```diff
@@ -12,4 +12,13 @@ export class NavbarComponent implements OnInit {
   ngOnInit() {
   }
 
+  isSessionTokenPresent() {
+    return localStorage.getItem('token') != null;
+  }
+
+  logout() {
```

Comment:
You can use router's API to redirect to the index/signin page rather than relying on browser's href.